index.html

```diff
@@ -45,5 +47,247 @@ <h1 class="presentation"> I'm Jorge E. Prada M.<br>
+                        hesitate to
+                        contact
+                        me.
+                    </p>
+                </div>
+                <div class="lets-connect">
+                    LET'S CONNECT
+                </div>
+                <ul class="icons-socialMedia2">
+                    <li class="iconnet iconnetMar">
+                        <a href="https://github.com/pradmon"><img src="./img/github.png" alt="github-icon"></a>                        
+                    </li>
+                    <li class="iconnet iconnetMar">
+                        <a href="https://www.linkedin.com/in/jorgepradamoncada/"><img src="./img/linkedin.png"
+                                alt="linkedin-icon"></a>
+                    </li>
+                    <li class="iconnet iconnetMar">
+                        <a href="https://www.angellist.com/"><img src="./img/angellist.png" alt="angellist-icon"></a>                        
+                    </li>
+                    <li class="iconnet iconnetMar">
+                        <a href="https://twitter.com/praduar"><img src="./img/twitter1.png" alt="twitter-icon"></a>
+                    </li>
+                    <li class="iconnet iconnetMar">
+                        <a href="https://medium.com/" rel="noopener noreferrer"><img src="./img/medium.png"
+                            alt="medium-icon"></a>
+                    </li>
+                </ul>
+                <div class="summary">
+                    <a class="text-decoration blue-color">
+                        <button class="btnsummary btn">Get my resume</button>
+                    </a>
+                </div>
+                <ul class="languageMar">
+                    <li class="iconnetMar">
+                        <div class="language">
+                            <div class="language-title">
+                                <div class="title1">Languages</div>
+                                <div class="arrow">
+                                    <img src="img/arrow.png" alt="arrow">
+                                </div>
+                            </div>
+                            <ul class="language-list marguin-list">
+                                <li class="list-flex ">
+                                    <a>
+                                        <img src="img/js.png" alt="JavaScript">
+                                    </a>
+                                    <div class="name-prom">
+                                        JavaScript
+                                    </div>
+                                </li>
+                                <li class="list-flex ">
+                                    <a>
+                                        <img src="img/html.png" alt="Html">
+                                    </a>
+                                    <div class="name-prom">
+                                        Html
```

Comment:
- **[OPTIONAL]** Please 🙏🏽  wrap the text of the div in a `p` tag for the best practices of HTML & CSS.